src/api/aptosConfig.ts

```diff
@@ -128,19 +183,52 @@ export class AptosConfig {
   }
 
   /**
-   * Checks if the URL is a known pepper service endpoint
-   *
-   * @internal
-   * */
+   * Checks if the provided URL is a known pepper service endpoint.
+   * 
+   * @param url - The URL to check against the known pepper service endpoints.
+   * 
+   * @example
+   * ```typescript
+   * import { Aptos, AptosConfig, Network } from "@aptos-labs/ts-sdk";
+   * 
+   * const config = new AptosConfig({ network: Network.TESTNET });
+   * const aptos = new Aptos(config);
+   * 
+   * async function runExample() {
+   *     const url = "https://example.pepper.service"; // replace with a real pepper service URL
+   * 
+   *     // Check if the URL is a known pepper service endpoint
+   *     const isPepperService = config.isPepperServiceRequest(url);
+   * 
+   *     console.log(`Is the URL a known pepper service? ${isPepperService}`);
+   * }
+   * runExample().catch(console.error);
+   * ```
+   */
   isPepperServiceRequest(url: string): boolean {
     return NetworkToPepperAPI[this.network] === url;
   }
 
   /**
-   * Checks if the URL is a known prover service endpoint
+   * Checks if the provided URL is a known prover service endpoint.
    *
-   * @internal
-   * */
+   * @param url - The URL to check against known prover service endpoints.
+   * @returns A boolean indicating whether the URL is a known prover service endpoint.
+   *
+   * @example
+   * ```typescript
+   * import { Aptos, AptosConfig, Network } from "@aptos-labs/ts-sdk";
+   *
+   * const config = new AptosConfig({ network: Network.TESTNET });
+   * const aptos = new Aptos(config);
```

Comment:
```suggestion
```

Reply:
Because AptosConfig is always used to create an `Aptos` instance I thought it was worth showing it creating that full class.